extensions/main.tf

```diff
@@ -96,10 +96,3 @@ resource "aws_ssm_parameter" "addons" {
   value = jsonencode(var.args.services)
   tags  = local.tags
 }
-
-module "logs" {
-  source = "../logs"
-
-  application = var.args.application
-  environment = var.environment
-}
```

Comment:
How come this has moved from extensions to vpc?

Reply:
So that we have one log resource policy per VPC instead of one per environment...

https://uktrade.atlassian.net/browse/DBTP-1089